Lib/ufo2ft/__init__.py

```diff
@@ -659,6 +666,10 @@ def compileVariableTTFs(designSpaceDoc: DesignSpaceDocument, **kwargs):
 
     logger.info("Building variable TTF fonts: %s", ", ".join(vfNameToBaseUfo))
 
+    if featuresNeeded:
+        # Skip generating feature variations in varLib.
```

Comment:
I've fixed this, but it's shown up that there's considerable code repetition between e.g. `compileVariableCFF2s` and `compileVariableTTFs`. I'm starting to think that these "compile a thing" functions may actually be better as *classes* rather than as *functions*. That would also help with the glyphsets plumbing problem (see comment below). So I might look into doing that major surgery.